RIPS/rip-7560.md

```diff
@@ -67,6 +67,26 @@ smart contracts. This, however, is impossible without an addition of Native Acco
 | MAX_CONTEXT_SIZE       | 65536               |
 | MAX_REVERT_REASON_SIZE | 1024                |
 
+
+### Definitions
+
+* *Transaction*:
+  An action initiated by an account and represented by a set of input parameters.
+  This action will change the state of the Ethereum blockchain when it is included in a block.
+* *RIP-7560 Transaction*:
+  An action initiated by a Smart Contract Account and represented with
+  an [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718) compatible Transaction Envelope object.
+* *RIP-7560 Execution Frame*:
+  A single atomic element of EVM code execution,
```

Comment:
a single call from an entryPoint address into the evm, to an RIP-7560 entity (account, paymaster or deployer).

Reply:
any evm call (either classic tx or 7560 frame) may contain inner "call frames"